js/yobit.js

```diff
@@ -758,10 +747,8 @@ module.exports = class yobit extends Exchange {
         //      }
         //
         const result = this.safeValue (response, 'return', {});
-        return {
-            'id': this.safeString (result, 'order_id'),
-            'info': result,
-        };
+        result['status'] = '2';
```

Comment:
We probably should not inject it here and find other ways to determine the status or leave it undefined:

```JavaScript
        const result = this.safeValue (response, 'return', {});
        return this.parseOrder (result);
```